pkg/sql/conn_executor_exec.go

```diff
@@ -548,6 +548,25 @@ func (ex *connExecutor) execStmtInOpenState(
 	p.noticeSender = res
 	ih := &p.instrumentation
 
+	if maxOpen := maxOpenTransactions.Get(&ex.server.cfg.Settings.SV); maxOpen > 0 {
```

Comment:
What is the behavior when `server.max_open_transactions_per_gateway` is zero? Seems like it's still unlimited?